manifests/params.pp

```diff
@@ -20,6 +20,25 @@
   $package_name       = 'vault'
   $package_ensure     = 'installed'
 
+  $checksum_type       = 'sha256'
+  $checksums           = {
```

Comment:
No it's more of a helper/maintenance thing, even if it just writes the checksums to stdout. Just imagine you need to bump the vault version, how do you now also update the checksums?